targets/pca10059.json

```diff
@@ -2,5 +2,5 @@
 	"inherits": ["nrf52840"],
 	"build-tags": ["pca10059"],
 	"linkerscript": "targets/pca10059.ld",
-	"flash-command": "nrfutil pkg generate --hw-version 52 --sd-req 0x0 --application {hex} --application-version 1 /tmp/tinygo_$$.zip && nrfutil dfu usb-serial -pkg /tmp/tinygo_$$.zip -p {port} -b 115200 && rm -f /tmp/tinygo_$$.zip"
+	"flash-command": "nrfutil dfu usb-serial -pkg {zip} -p {port} -b 115200"
```

Comment:
The name zip is probably too common and will cause problems later.
I think the name `nrf-dfu`, as written by @aykevl , is fine.